### PR TITLE
📦 NEW: Domain App

### PR DESCRIPTION
**Pull request by ahmadbilaldev**
This PR adds a [Domain Search App](https://rapidapi-example-domain-app.vercel.app/) built using a WHOIS API from RapidAPI Hub.

@ghoshnirmalya kindly take a look.

### CODE REVIEW

**Review by ghoshnirmalya — COMMENTED** [review]
I'm getting a 504 status code.

Screenshot 2021-12-19 at 12 59 58 PM

**Comment by ahmadbilaldev** [comment]
> I'm getting a 504 status code.

The API requires the domain name to be valid i.e., `godaddy.com` instead of just `godaddy`. I added error handling to inform the user about this.

File: domain-app/pages/api/search.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+  };
+
+  axios
+    .request(options)
+    .then(function (response) {
+      res.status(200).json(response.data);
+    })
+    .catch(function (error) {
+      console.error(error);
```
Comment (ghoshnirmalya) [inline domain-app/pages/api/search.js:19]:
```suggestion
```

**Review by ghoshnirmalya — CHANGES_REQUESTED** [review]
The form should submit even when enter is pressed.

**Review by ghoshnirmalya — APPROVED** [review]

**Pull request merged**